lib/pacemaker/pcmk_sched_primitive.c

```diff
@@ -1417,6 +1417,15 @@ pcmk__primitive_add_graph_meta(pe_resource_t *rsc, xmlNode *xml)
                         parent->container->id);
         }
     }
+
+    /* Bundle replica children will get their external-ip set internally as a
+     * meta-attribute. The graph action needs it, but under a different naming
+     * convention than other meta-attributes.
+     */
```

Comment:
Where is this even used? I haven't been able to find any references to it yet outside of setting it in `bundle.c`, getting it in `pcmk_graph_producer.c` to add to the graph attrs XML, and filtering it out in `operations.c`... I don't see where it's actually parsed out of the graph XML and used. Tried to be pretty generous with my grepping

Reply:
I believe it's set for resource agent use

Reply:
Maybe. There are no resource agents using it in resource-agents or pacemaker.

Reply:
It would be the OpenStack-related ones